api/app/stations.py

```diff
@@ -71,12 +71,12 @@ def _set_weather_variables(station_properties: DetailedWeatherStationProperties,
             setattr(weather_variables, variable_name, value)
 
 
-def _get_detailed_stations(time_of_interest: datetime):
+async def _get_detailed_stations(time_of_interest: datetime):
```

Comment:
Looks like SonarCloud complains about no tests for this function, but all that was changed is sync -> async and where it gets stations from. I believe this is tested by the endpoint tests implicitly though, right?

Reply:
yes, it's implicitly tested because we have at least 1 test for the /stations/details endpoint